bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java

```diff
@@ -621,17 +622,27 @@ void drawSelectedTab(int itemIndex, GC gc, Rectangle bounds) {
 		}
 
 		if (selectedTabHighlightColor != null) {
+			Color oldBackground = gc.getBackground();
```

Comment:
Why do we (re-)store the background color. This was not necessary before and I don't see that this changed.